.coafile

```diff
@@ -0,0 +1,9 @@
+[all]
+bears = JSONFormatBear
+indent_size = 2
+files = **.json 
+
+[whitespace]
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/.coafile
+++ b/.coafile
@@ -1,7 +1,7 @@
 [all]
 bears = JSONFormatBear
 indent_size = 2
-files = **.json 
+files = **.json
 
 [whitespace]
 bears = SpaceConsistencyBear
```